salt/transport/zeromq.py

```diff
@@ -728,6 +737,8 @@ def publish(self, load):
             int_payload['topic_lst'] = load['tgt']
 
         pub_sock.send(self.serial.dumps(int_payload))
+        pub_sock.close()
```

Comment:
Maybe set a linger to 5? ie, wait 5 seconds before dropping anything being sent an close?